python/generate-short-url/README.md

```diff
@@ -0,0 +1,61 @@
+# 📱 Validate phone number and get it's country information
+
+A Python Cloud Function that generates short URLs with help of Bitly or Tinyurl.
+
+_Example input:_
```

Comment:
Let's please have one more example where we showcase payload with provider `bitly`. To make sure it' clear these two providers are supported.